tests/renderTopLanguages.test.js

```diff
@@ -216,23 +216,23 @@ describe("Test renderTopLanguages", () => {
     );
     expect(queryAllByTestId(document.body, "lang-progress")[0]).toHaveAttribute(
       "width",
-      "120",
+      "100",
     );
 
     expect(queryAllByTestId(document.body, "lang-name")[1]).toHaveTextContent(
       "javascript 40.00%",
     );
     expect(queryAllByTestId(document.body, "lang-progress")[1]).toHaveAttribute(
       "width",
-      "120",
+      "100",
     );
```

Comment:
Why are you reducing this by `20` (and `10` below)? Shouldn't you reduce by `50` since that's what the changed code does?

Reply:
Let me share the math behind it. So the `DEFAULT_CARD_WIDTH = 300`

### Previously 
card_width = 350 (extra 50px)
language_bar_width = 300
HTML is 40% of 300 means 120.
javascript is 40% of 300 means 120.
CSS is 20% of 300 means 60.

### NOW
card_width = 300 (no extra 50px)
language_bar_width = 250 (because of 25px padding on either side)
HTML is 40% of 250 means 100.
javascript is 40% of 250 means 100.
CSS is 20% of 250 means 50.

Reply:
Oh, I misunderstood the code; my bad! Makes sense now.